src/mongo/ModifiedSubject.class.php

```diff
@@ -85,8 +85,8 @@ public static function create(Array $resourceId, Array $types, Array $operations
         $data = array(
             _ID_RESOURCE=>$resourceId[_ID_RESOURCE],
             _ID_CONTEXT=>$resourceId[_ID_CONTEXT],
-            "database"=>$dbName,
-            "collection"=>$collectionName,
+            "database"=>$storeName,
+            "collection"=>$podName,
```

Comment:
The 'database' key was removed, since it's no longer necessary, but I kept the keyname 'collection' so we don't have to worry about queued documents being incompatible when we upgrade tripod.